tango_simlib/utilities/helper_module.py

```diff
@@ -123,10 +118,10 @@ def cmd_handler(tango_device, input_parameters=None):
         cmd_info_copy = model.sim_actions_meta[action_name].copy()
         # Delete all the keys that are not part of the Tango command parameters.
         cmd_info_copy.pop('name')
-        tango_cmd_prop = POGO_USER_DEFAULT_CMD_PROP_MAP.values()
+        tango_cmd_props = DEFAULT_CMD_PROPS
         for prop_key in model.sim_actions_meta[action_name]:
-            if prop_key not in tango_cmd_prop:
-                MODULE_LOGGER.info(
+            if prop_key not in tango_cmd_props:
```

Comment:
Could just iterate through `DEFAULT_CMD_PROPS` instead of assigning another variable, but no matter.